gen.py

```diff
@@ -113,16 +113,47 @@ class Vars(BaseModel):
     __root__: List[Union[FilePath, VarDecl]]
 
 
+STAGE_VARS_DESC = """\
+Sequence of variables for parametrization.
+
+May include any mapping or a path to a params file
+(which may be a string or a mapping to params in the file).
+
+Use in the stage with the `${}` substitution expression."""
+
+CMD_DESC = """\
+(Required) Command to run (anything your system terminal can run).
+Can be a string or a sequence of commands."""
+
+PARAMS_DESC = """\
+Sequence of dot-separated parameter dependency keys to track from `params.yaml`.
```

Comment:
VS Code doesn't consume these backticks in the expected way:

![image](https://user-images.githubusercontent.com/37993418/189552867-496aec86-e9ae-41ab-a3fb-933e9eaeca61.png)

Reply:
Yeah, it's not going to generate code blocks. Should we replace them with something else? I'm not sure if there's a better way to mark them or if it's fine for them to show as backticks. @jorgeorpinel Maybe you have some thoughts here?

Reply:
Regular `'` or nothing? Can you use HTML? I.e. `<code>` tags